elementary/monitor/dbt_project/macros/base_queries/current_tests_run_results_query.sql

```diff
@@ -1,23 +1,21 @@
-{% macro current_tests_run_results_query(days_back = none, invocation_id = none) %}
+{% macro current_tests_run_results_query(days_back = none) %}
```

Comment:
Why was this removed?
Isn't it necessary for the filters?

Reply:
We do the filter now on the python level.
Instead of query the same data 3 times (1 for test results, 1 for invocations, 1 for data samples) we now query it only once, and filter the data when needed on the relevant method.

Invocation is not filtering the invocation part (test runs), so we need to do this filter outside of the query.

Reply:
Awesome!